Plugins/Creature/NWScript/nwnx_creature.nss

```diff
@@ -2355,3 +2360,12 @@ int NWNX_Creature_GetBypassEffectImmunity(object oCreature, int nImmunityType)
     NWNX_CallFunction(NWNX_Creature, sFunc);
     return NWNX_GetReturnValueInt();
 }
+
+void NWNX_Creature_SetLastKiller(object oCreature, object oKiller)
+{
+    string sFunc = "SetLastKiller";
+
+    NWNX_PushArgumentObject(NWNX_Creature, sFunc, oKiller);
+    NWNX_PushArgumentObject(NWNX_Creature, sFunc, oCreature);
```

Comment:
The arguments for `NWNX_{Push|Pop}Argument{Type}` have changed, they no longer take the Plugin/Function name.

Eg:

```
NWNX_PushArgumentObject(oKiller);
NWNX_PushArgumentObject(oCreature);
```    

Don't forget to update your own nwnx.nss and other nwnx_*.nss scripts!